### PR TITLE
Updates Staff of Change pool

### PR DESCRIPTION
**Pull request by ShadowDeath6**
Moves the nonharmful mobs like mushrooms, bats, and goats to the animals section and adds some new things, such as syndie medborg and honkbot

sorry for the admins.txt config change but my branches have been really fucky and I don't think changing the config actually changes anything, let me know if I'm wrong.

:cl: ShadowDeath6
rscadd: Updated the Staff of Change transformation pool.
/:cl:

### CODE REVIEW

**Comment by oisin100** [comment]
Add borers to staff of change when?

**Comment by ShadowDeath6** [comment]
That's actually a good one tbh

File: config/admins.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@ Slovenian	=	Council Member
 George99	=	Council Member
 Thee_Flame_Prince	=	Council Member
 Xantam	=	Council Member
-AsV9	=	Council Member
+ShadowDeath6	=	Council Member
```
Comment (Super3222) [inline config/admins.txt:16]: You forgot something.

Comment (ShadowDeath6) [reply config/admins.txt:16]: read the OP

**Pull request merged**